tests/CMakeLists.txt

```diff
@@ -64,7 +78,6 @@ function(add_aktualizr_test)
                      ${WD})
         endif()
     add_dependencies(build_tests t_${AKTUALIZR_TEST_NAME})
-    aktualizr_source_file_checks(${AKTUALIZR_TEST_SOURCES})
```

Comment:
What about adding an option to `add_aktualizr_test` to skip adding the sources?  That would remove one more place for people to forget. Adding the .h files is a good catch, though

Reply:
Possible, but this gets tricky fast. The problematic file is fake_ostree.cc. It is used as a supplemental file by two tests, both of which also have an individual .cc as well. Hence, if you skipped adding the sources of one, you'd still have to manually add the individual .cc.

But I was sad to have to break your logic there, so maybe that is worth it. (Or is there a way to skip duplicates? Or could `add_aktualizr_test` maybe just add the files to a list, which can be de-dup'd, and then run `aktualizr_source_file_checks` on the list at the end?)